tests/js/server/aql/aql-graph.js

```diff
@@ -1646,6 +1672,68 @@ function kPathsTestSuite() {
   };
 }
 
+function allShortestPathsTestSuite() {
+  const gn = "UnitTestGraph";
+  const vn = "UnitTestV";
+  const en = "UnitTestE";
+
+  return {
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set up
+////////////////////////////////////////////////////////////////////////////////
+
+    setUpAll: function () {
+      gm._create(gn, [gm._relation(en, vn, vn)]);
+
+      ["s", "t", "a", "b", "c", "d", "e", "f", "g"].map((elem) => {
+        db[vn].insert({_key: elem});
+      });
+
+      [
+        ["s", "a"], ["s", "b"], ["a", "b"], ["a", "c"], ["b", "c"],
+        ["c", "d"], ["d", "t"],
+        ["c", "e"], ["e", "t"],
+        ["c", "f"], ["f", "t"],
+        ["c", "g"], ["g", "t"]
```

Comment:
To make the "ANY" test different.
I would suggest to add one or two "wrong direction" edges to this graph.
Right now all paths flow from S to T.

For any we need a path like this:

S -> X <- Y <- Z -> T
which should be found
If we have another edge Y -> Z in above example, ANY should give two paths:
S -> X <- Y <- Z -> T (as above)
S -> X <- Y -> Z -> T (use the other edge from Y to Z)

Reply:
added! thank you for that

addressed in 7dd39c35d94